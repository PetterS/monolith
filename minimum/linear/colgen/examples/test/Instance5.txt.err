Number of days: 28
-- Shift E, 480 min.
-- Shift L, 480 min.
Number of shifts: 2
-- Shift 0. OK after = [0, 1] forbidden_after = []
-- Shift 1. OK after = [1] forbidden_after = [0]
-- Staff  A: Consequtive limit [2, 5]. Time limit [7560, 8640]. Max weekends: 2. Max shifts: [28, 0].
-- Staff  B: Consequtive limit [2, 5]. Time limit [7560, 8640]. Max weekends: 2. Max shifts: [28, 0].
-- Staff  C: Consequtive limit [2, 5]. Time limit [7560, 8640]. Max weekends: 2. Max shifts: [28, 14].
-- Staff  D: Consequtive limit [2, 5]. Time limit [7560, 8640]. Max weekends: 2. Max shifts: [28, 14].
-- Staff  E: Consequtive limit [2, 5]. Time limit [7560, 8640]. Max weekends: 2. Max shifts: [28, 0].
-- Staff  F: Consequtive limit [2, 5]. Time limit [7560, 8640]. Max weekends: 2. Max shifts: [28, 14].
-- Staff  G: Consequtive limit [2, 5]. Time limit [7560, 8640]. Max weekends: 2. Max shifts: [28, 14].
-- Staff  H: Consequtive limit [2, 5]. Time limit [7560, 8640]. Max weekends: 2. Max shifts: [28, 14].
-- Staff  I: Consequtive limit [2, 5]. Time limit [7560, 8640]. Max weekends: 2. Max shifts: [28, 14].
-- Staff  J: Consequtive limit [2, 5]. Time limit [7560, 8640]. Max weekends: 2. Max shifts: [28, 14].
-- Staff  K: Consequtive limit [2, 6]. Time limit [7560, 8640]. Max weekends: 3. Max shifts: [28, 0].
-- Staff  L: Consequtive limit [2, 6]. Time limit [7560, 8640]. Max weekends: 3. Max shifts: [28, 0].
-- Staff  M: Consequtive limit [2, 6]. Time limit [7560, 8640]. Max weekends: 3. Max shifts: [28, 14].
-- Staff  N: Consequtive limit [2, 6]. Time limit [7560, 8640]. Max weekends: 3. Max shifts: [28, 0].
-- Staff  O: Consequtive limit [2, 6]. Time limit [7560, 8640]. Max weekends: 3. Max shifts: [28, 14].
-- Staff  P: Consequtive limit [2, 6]. Time limit [7560, 8640]. Max weekends: 3. Max shifts: [28, 14].
Number of staff: 16
Read 79 shift on requests.
Read 27 shift off requests.
Read 56 cover constraints.
Dividing all minutes by 480
Setting up colgen problem...[  OK  ]
Creating initial rosters...[  OK  ]
-- Objective for initial columns: 5159
  Iter |       Objective      |    Problem   |    Pool   | Fixed  |                  Time                  | Frac. |
       |    frac.     rounded |   size  diff | size gen. |        |    fix       gen     solve   tot. cum. |       |
      1       5159       5159        0  128     128    0      0        n/a       n/a     n/a     n/a          0%
      2     3288.2       3550       35   35     163   35      0        n/a       n/a     n/a     n/a          7%
      3     2600.4       3925       63   28     193   30      0        n/a       n/a     n/a     n/a          2%
      4    2242.86       2619       97   34     228   35      0        n/a       n/a     n/a     n/a          4%
      5    1936.48       2796      120   23     255   27      0        n/a       n/a     n/a     n/a          9%
      6    1802.94       2300      144   24     286   31      0        n/a       n/a     n/a     n/a          7%
      7    1671.22       2182      153    9     310   24      0        n/a       n/a     n/a     n/a          8%
      8    1534.15       2510      166   13     343   33      0        n/a       n/a     n/a     n/a          6%
      9    1372.05       2917      160   -6     363   20      0        n/a       n/a     n/a     n/a          6%
     10    1327.36       2205      164    4     402   39      0        n/a       n/a     n/a     n/a          6%
     11    1273.85       2404      171    7     427   25      0        n/a       n/a     n/a     n/a          4%
     12    1243.51       2505      170   -1     452   25      0        n/a       n/a     n/a     n/a          4%
     13    1231.47       2305      161   -9     486   34      0        n/a       n/a     n/a     n/a          7%
     14    1201.63       2199      154   -7     506   20      0        n/a       n/a     n/a     n/a          9%
     15    1197.79       2299      143  -11     529   23      0        n/a       n/a     n/a     n/a          3%
     16    1187.98       2097      127  -16     544   15      0        n/a       n/a     n/a     n/a          7%
     17       1185       2299      125   -2     576   32      0        n/a       n/a     n/a     n/a          6%
     18    1178.89       2088      131    6     602   26      0        n/a       n/a     n/a     n/a          5%
     19    1176.38       2088      133    2     631   29      0        n/a       n/a     n/a     n/a          3%
     20    1175.55       1886      125   -8     654   23      0        n/a       n/a     n/a     n/a          5%
-- Fixed for [A: 18, B: 16, C: 7, D: 17, E: 12, F: 18, G: 15, H: 14, I: 7, J: 18, K: 17, L: 16, M: 8, N: 16, O: 10, P: 17] at level 0.75.
-- Members remaining: (C, 3) (E, 2) (H, 3) (I, 2) (K, 2) (M, 2) (N, 2) (O, 4) and total is 8.
     21    1555.42       1984       37  -88     158    4    500        n/a       n/a     n/a     n/a          2%
     22    1401.23       1778       45    8     165    7      0        n/a       n/a     n/a     n/a          6%
     23     1377.4       1475       50    5     173    8      0        n/a       n/a     n/a     n/a          8%
     24     1367.2       1475       55    5     177    4      0        n/a       n/a     n/a     n/a          7%
     25    1362.57       1475       55    0     180    3      0        n/a       n/a     n/a     n/a          4%
     26     1352.5       1576       51   -4     182    2      0        n/a       n/a     n/a     n/a          1%
     27     1351.5       1475       52    1     185    3      0        n/a       n/a     n/a     n/a          4%
-- Fixed for [C: 4, E: 3, G: 3, H: 4, I: 9, K: 1, M: 9, N: 2, O: 5] at level 0.75.
-- Members remaining: (C, 3) (E, 2) (I, 2) (O, 2) and total is 4.
     28       1275       1275       26  -26     139    3     49        n/a       n/a     n/a     n/a          0%
     29       1275       1275       29    3     143    4      0        n/a       n/a     n/a     n/a          0%
-- Stopping. Objective change is 0 with an integer feasible solution.
Colgen done.
Elapsed time : n/a.
  Iter |       Objective      |    Problem   |    Pool   | Fixed  |                  Time                  | Frac. |
       |    frac.     rounded |   size  diff | size gen. |        |    fix       gen     solve   tot. cum. |       |
      1       1275       1275       29    0     692    0      0        n/a       n/a     n/a     n/a          0%
      2    1245.08       1375      185  156     728   36      0        n/a       n/a     n/a     n/a          0%
      3    1171.55       1778      332  147     751   23      0        n/a       n/a     n/a     n/a          3%
      4    1165.17       1878      359   27     776   25      0        n/a       n/a     n/a     n/a          4%
      5    1164.59       1876      378   19     796   20      0        n/a       n/a     n/a     n/a          3%
-- Fixed for [A: 18, B: 16, C: 11, D: 12, E: 9, F: 18, G: 6, H: 18, I: 7, J: 11, K: 14, L: 16, M: 17, N: 17, O: 10, P: 6] at level 0.75.
-- Members remaining: (C, 2) (D, 2) (E, 2) (G, 2) (I, 3) (J, 2) (K, 2) (N, 2) (O, 3) (P, 2) and total is 10.
      6     1368.5       1773       48 -330     179    8    625        n/a       n/a     n/a     n/a          8%
      7    1301.67       1878       64   16     188    9      0        n/a       n/a     n/a     n/a          8%
      8    1293.09       1873       67    3     197    9      0        n/a       n/a     n/a     n/a          3%
      9    1267.21       1369       77   10     205    8      0        n/a       n/a     n/a     n/a          1%
     10       1264       1366       78    1     212    7      0        n/a       n/a     n/a     n/a          3%
     11    1263.55       1464       81    3     219    7      0        n/a       n/a     n/a     n/a          0%
-- Fixed for [C: 1, D: 3, E: 9, G: 8, I: 6, K: 2, O: 7, P: 9] at level 0.75.
-- Members remaining: (C, 2) (D, 2) (G, 2) (I, 2) (J, 4) (P, 2) and total is 6.
     12       1266       1266       38  -43     155    5     69        n/a       n/a     n/a     n/a          0%
     13       1264       1264       45    7     161    6      0        n/a       n/a     n/a     n/a          0%
     14       1264       1264       48    3     166    5      0        n/a       n/a     n/a     n/a          0%
-- Stopping. Objective change is 0 with an integer feasible solution.
Colgen done.
Elapsed time : n/a.
Solution objectives: [1275, 1264].
