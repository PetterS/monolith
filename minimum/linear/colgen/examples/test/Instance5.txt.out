  A:  . .E=E=E= . .E=E=E=E=E= . .E=E=E= . .E=E= . .E=E=E=E=E=
  B: E=E=E= . .E=E=E=E= . . . . .E=E=E=E=E= . . . . .E=E=E=E=
  C: E=E=L=L= . . . . .L=L=L=L=L= . .E=E=E= . .L=L=L= . .L=L=
  D:  . . .L=L=L=L=L= . .E=L= . .E=E=L=L=L= . .L=L=L= . .E=E=
  E: E=E=E=E=E= . .E=E=E= . .E=E=E=E=E= . . . . . . .E=E=E=E=
  F: L=L=L= . .E=E=L=L=L= . . . .L=L=L=L=L= . .E=E=L= . .L=L=
  G: L=L= . .E=L=L= . .E=L=L= . .L=L=L=L=L= . .L=L=L= . .E=E=
  H: E=E=E=E=E= . .E=L=L= . .L=L= . .E=E=E=L=L= . .L=L=L= . .
  I: L=L=L= . .E=E=L=L=L= . .L=L=L=L=L= . . . .E=E=E=E=L= . .
  J: L=L=L=L=L= . . .E=L=L=L= . . . . .E=E=E=L= . .E=L=L=L=L=
  K:  .E=E=E=E= . . .E=E=E=E=E=E= . . .E=E=E=E=E= . . .E=E=E=
  L:  . . .E=E=E=E=E= . . .E=E=E=E=E= . . .E=E=E=E=E=E= . . .
  M:  . .L=L=L= . . .E=E=E=E=E=E= . . .E=E=L=L=L=L= . . .L=L=
  N:  .E=E=E=E=E=E= . . .E=E=E=E=E=E= . . .E=E=E=E=E=E= . . .
  O: E=E= . . .E=E=E=L=L=L= . . . .L=L=L=L=L=L= . . .L=L=L=L=
  P:  . .L=L=L=L=L=L= . . .E=E=E=E=E= . . .E=E=E=E=E=L= . . .
Final solution (checked for feasibility): 1275
-- Cover costs  : 1203
-- Roster costs : 72
  A:  .E=E=E=E= . .E=E=E=E=E= . . .E=E= . .E=E= . .E=E=E=E=E=
  B: E=E=E= . .E=E=E=E= . . . . .E=E=E=E=E= . . . . .E=E=E=E=
  C: E=E=L=L=L= . . . .L=L=L=L=L= . .E=E=E= . .E=E=E= . .E=E=
  D: L=L=L= . .L=L=L= . .E=L= . .E=E=E=L=L= . .L=L=L= . .E=E=
  E: E=E=E=E=E= . .E=E=E= . .E=E=E=E=E= . .E=E=E= . .E=E= . .
  F: L=L=L= . .E=E=L=L=L= . . . .L=L=L=L=L= . .E=E=L= . .L=L=
  G: L=L= . .E=L=L= . .E=L=L= . .L=L=L=L=L= . .L=L=L= . .E=E=
  H: E=E=E=E=E= . .E=L=L= . .L=L= . .E=E=E=L=L= . .L=L=L= . .
  I: L=L= . . .E=E=L=L=L= . .L=L=L=L=L= . . . .E=E=E=L=L= . .
  J: E=E=L=L= . . .E=E=L=L=L= . . . . .E=E=E=L= . .L=L=L=L=L=
  K:  . . .E=E= . . .E=E=E=E=E=E= . . .E=E=E=E= . . .E=E=E=E=
  L:  . . .E=E=E=E=E= . . .E=E=E=E=E= . . .E=E=E=E=E=E= . . .
  M:  . .L=L=L= . . .E=E=E=E=E=E= . . .E=E=L=L=L=L= . . .L=L=
  N:  .E=E=E=E=E=E= . . . .E=E=E=E=E= . . .E=E=E=E=E=E= . . .
  O:  . . . . .E=L=L=L=L=L= . . .E=L=L=L=L=L= . . .E=L=L=L=L=
  P:  . .L=L=L= . . . .E=E=E=E=E=E= . . .E=L=L=L=L= . . .L=L=
Final solution (checked for feasibility): 1264
-- Cover costs  : 1200
-- Roster costs : 64
